extensions/markdown-language-features/package.nls.json

```diff
@@ -22,6 +22,8 @@
 	"markdown.preview.refresh.title": "Refresh Preview",
 	"markdown.preview.toggleLock.title": "Toggle Preview Locking",
 	"markdown.findAllFileReferences": "Find File References",
+	"markdown.editor.insertLinkToLocalFile": "Insert link to local file",
+	"markdown.editor.insertLocalImage": "Insert local image",
```

Comment:
It looks like this would work in VS Code for the web, but 'local' suggests desktop-only--wondering if the 'local' in the command name is really necessary (I'm guessing it means that you can't insert links across different filesystems)?

Reply:
I've updated these to say `from workspace` instead. This hopefully make it clear that we support selecting things on the same FS as your current workspace and can't for example insert an image from your desktop into a web instance of VS Code (at least with our current apis)